pkg/dogstatsd/server.go

```diff
@@ -341,6 +345,41 @@ func NewServer(demultiplexer aggregator.Demultiplexer, serverless bool) (*Server
 		}
 	}
 
+	// Creating a new logger
+	cfg := NewSeelogConfig(
+		"dogstatsd",
+		"info",
+		"common",
+		"",
+		"%n %Date %Time | [DOGSTATSD_STATSD] | [%LEV] %Msg %n",
```

Comment:
```suggestion
		"%n %Date %Time | [DOGSTATSD_STATS] | [%LEV] %Msg %n",
```

Reply:
Also can remove the leading `%n` as that is adding a blank line between every log line in the file. 
[Format docs are here for reference ](https://github.com/cihub/seelog/wiki/Format-reference)